### ISSUE TITLE
Support for @angular/cli v14 for generator

### ISSUE THREAD

**Issue by marcin-wosinek** [bgotink/ngx-playwright#5]
I'm working on setting up a simple angular + playwright project. While trying to set it up with this project, I got the following errors:
```
// my main project
$ ng generate @ngx-playwright/test:setup --project ng-14
Project "ng-14" is not an application project

// wrong name for comparizion
$ ng generate @ngx-playwright/test:setup --project lorem-ispum
Project "lorem-ispum" doesn't exist in the workspace
```

In 'package.json' I see `@angular/cli` at v13—I guess there are some differences in schema that cause the install to fail.

**Comment by marcin-wosinek** [comment]
I tried with application generated with cli v13 and the result is the same.

**Comment by bgotink** [comment]
Looks like the schematic checks for the nonexistent `type` property to look for applications, while it should check for `projectType`.

I've also turned the error into a warning when you explicitly pass the `--project` flag, in case `projectType` is not set (I know I more or less never set it).

**Comment by bgotink** [comment]
Should be fixed in version 0.2.6!

**Comment by marcin-wosinek** [comment]
Cool, thanks a lot!
